CHANGELOG.md

```diff
@@ -4,7 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [2.2.0](https://github.com/ASFHyP3/hyp3/compare/v2.2.1...v2.2.0)
+## [2.2.1](https://github.com/ASFHyP3/hyp3/compare/v2.2.0...v2.2.1)
```

Comment:
I expect this will be either part of the 2.2.0 release along with subscriptions, or possibly a 2.1.2 release prior to subscriptions

Reply:
Fair, lets do 2.1.2, I think the workflow would be

Change the changelog, removing the subscriptions entry (2.2.0), merge to develop, restore branch, merge to main, re add subscriptions entry

Reply:
actually I've changed my mind, lets send it with subscriptions